engines/python/src/main/java/ai/djl/python/engine/PyProcess.java

```diff
@@ -237,6 +240,10 @@ boolean isStopped() {
         return !started;
     }
 
+    boolean isModelLoaded() {
```

Comment:
We can combine isStopped() and isModelLoaded() into `isRead()`

Reply:
done